Blockchain/Foundational topics/Distributed systems.md

```diff
@@ -0,0 +1,70 @@
+---
+tags: blockchain, foundational topics, distributed systems
```

Comment:
Tag doesn't work with space. If the tag contains more than 1 word, we should use `-` to join them.

```suggestion
tags: blockchain, foundational-topic, distributed-system
```